openage/testing/benchmark.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017-2017 the openage authors. See copying.md for legal info.
+
+""" Benchmarking tools for the tests. """
+
+from timeit import timeit
+from sys import stdout
+from time import sleep
+
+
+def benchmark():
+    """ Simple function to call in for benchmarking. """
+    sleep(0.1)
+
+
+def benchmark_func(func):
+    """
+    Benchmark the given function. Repeated execution helps to give a maximum to
+    the consumed time, until one iteration takes more than 5s, summed up 10s.
+    """
+    result = [(0, 1)]
+    number = 1
+    total = [0, 0]
+    str_row_format = "{:10} {:12}  {:11}"
+    row_format1 = "{:10} "
+    row_format2 = "{:11.8f}s  {:10.8f}s"
```

Comment:
This is there, that I can just do ```row_format1+row_format2``` for the output in the end, and I don't have to change that many locations.